### PR TITLE
add ArbitraryAttributes parameter (#274)

### LINKED ISSUE #274: Capture other html attributes
It is a best practice to allow the component to capture other HTML element attributes that are not specified in Blazor code.
maybe through a Parameter with (CaptureUnmatchedValues = true) which could be named "Attributes".

### PR DESCRIPTION
**Pull request by maryamhdr**
this closes #274

### CODE REVIEW

**Comment by ysmoradi** [comment]
Finally!
It was missing
Anyway, shall we write at least one unit test for this? for example using BitButton and arbitrary attributes?
@maryamhdr @msynk

**Comment by msynk** [comment]
Unfortunately, It seems this implementation does not work!
It might be because of the difference between the definition location of the property vs the usage location of it and how the SetParametersAsync is overridden in this project.

**Review by msynk — CHANGES_REQUESTED** [review]
please provide some unit test for this new feature and also some demo implementations in Palygound project

**Comment by msynk** [comment]
let's move writing appropriate unit tests for this feature to another issue.

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitChoiceOption.razor.cs [hunk 1]
```diff
@@ -34,41 +35,49 @@ public bool IsChecked
 
         public override Task SetParametersAsync(ParameterView parameters)
```
[the remark this answers is not in the document]
Comment (msynk) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Inputs/BitChoiceOption.razor.cs:36]: I'm with the investigation and we can safely remove this method completely

**Review by msynk — APPROVED** [review]

**Pull request merged**